### DISCUSSION TITLE
can't access deta drive & empty drive doesn't auto-delete

### DISCUSSION THREAD

**Discussion by jnsougata** [deta/cloud-docs#283] [Help] [answered]
![image](https://user-images.githubusercontent.com/53375272/174512391-bf54b37a-7c2d-408a-8b45-439376f30c4d.png)
 
I'm unable to access this drive or delete this as I can't get inside. Even more this drive doesn't contain of any files still existing. 
I thinks this behavior just for the `#` in the name of the drive. Project ID: `c0l0ezfh` TIA

**Comment by abdelhai** [comment, marked as answer]
i believe this is the issue. i will report it and see if we can manually change the name. for now avoid using `#` in names.
